packages/lib/package.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
what should go under this `lib` package?

Reply:
Right now, it's serving as, more or less, client/server-agnostic libraries.